### PR TITLE
lightning: precheck should check CDC changefeeed rather than capture

### LINKED ISSUE #41040: lighting import with local backend failed if target cluster has CDC node
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
1. Deploy TiDB cluster with CDC node （do not create CDC changefeed）
2. Use lightning to import data to cluster, using local backend

### 2. What did you expect to see? (Required)
Lightning import should succeed

### 3. What did you see instead (Required)
Lightning import precheck failed due to  "found CDC capture"
cfdea5c5-c797-4531-8d3c-2d3cb1201862

### 4. What is your TiDB version? (Required)
[root@lightning-0 /]# /tidb-lightning -V
Release Version: v6.6.0-alpha
Git Commit Hash: 6e56bf717c1b637fd65af618e0ee36cbad568953
Git Branch: heads/refs/tags/v6.6.0-alpha
Go Version: go1.19.5
UTC Build Time: 2023-02-02 11:43:38
Race Enabled: false

### PR DESCRIPTION
**Pull request by lance6716**
Signed-off-by: lance6716 <lance6716@gmail.com>

### What problem does this PR solve?

Issue Number: close #41040

Problem Summary:

### What is changed and how it works?

change check logic introduced in https://github.com/pingcap/tidb/pull/39338

as title

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/lightning/restore/precheck_impl.go [hunk 1]
```diff
@@ -780,49 +781,59 @@ func (ci *CDCPITRCheckItem) Check(ctx context.Context) (*CheckResult, error) {
 
 	// check etcd KV of CDC >= v6.2
 	cdcPrefix := "/tidb/cdc/"
-	capturePath := []byte("/__cdc_meta__/capture/")
+	capturePath := []byte("/changefeed/info/")
```
Comment (asddongmen) [inline br/pkg/lightning/restore/precheck_impl.go:784]: typo

**Review by asddongmen — APPROVED** [review]

File: br/pkg/lightning/restore/precheck_impl.go (hunk 1 above)
Comment (lance6716) [reply br/pkg/lightning/restore/precheck_impl.go]: https://github.com/pingcap/tidb/pull/41076/commits/099535ec6c2db0d90a6970c4de6771a7259f8b3d

**Review by sdojjy — APPROVED** [review]

**Pull request merged**